packages/boxel/addon/components/boxel/tab-bar/index.css

```diff
@@ -19,13 +52,12 @@
   https://css-tricks.com/bold-on-hover-without-the-layout-shift/
 */
 
-.boxel-tab-bar__item > a {
-  display: inline-flex;
+.boxel-tab-bar__item-text {
+  display: flex;
   flex-direction: column;
-  padding: var(--boxel-sp-xs) var(--boxel-sp-lg);
 }
 
-.boxel-tab-bar__item > a::after {
+.boxel-tab-bar__item-text::after {
```

Comment:
`font-weight` for this should be `var(--boxel-tab-bar-font-weight-hover)`, since we allow configuring it.